docs/sql-reference/00-sql-reference/51-connect-parameters.md

```diff
@@ -137,4 +137,16 @@ The following table lists connection parameters for accessing WebHDFS:
 | Parameter    	| Required? 	| Description                                       	|
 |--------------	|-----------	|---------------------------------------------------	|
 | endpoint_url 	| Yes       	| Endpoint URL for WebHDFS.                         	|
-| delegation   	| No        	| Delegation token for accessing WebHDFS.           	|
\ No newline at end of file
+| delegation   	| No        	| Delegation token for accessing WebHDFS.           	|
+
+
+### Hugging Face
+
+The following table lists connection parameters for accessing Hugging Face:
+
+| Parameter | Required?             | Description                                                                                                     |
+|-----------|-----------------------|-----------------------------------------------------------------------------------------------------------------|
+| repo_id   | Yes                   | The identifier for the Hugging Face repository. For example, "opendal/huggingface-testdata".                    |
```

Comment:
Please add this note: We only support `repo_id` with org names, datasets like https://huggingface.co/datasets/ropes is not supported yet.

Reply:
Please note that the repo_id must have an organization name; datasets (such as https://huggingface.co/datasets/ropes) stored in a non-organizational format on the Hugging Face, are not supported at this time.